ildcp-core/src/main/java/org/interledger/ildcp/IldcpRequestPacket.java

```diff
@@ -2,24 +2,29 @@
 
 import org.interledger.annotations.Immutable;
 import org.interledger.core.InterledgerAddress;
+import org.interledger.core.InterledgerAddressPrefix;
 import org.interledger.core.InterledgerCondition;
 import org.interledger.core.InterledgerPreparePacket;
 
+import org.immutables.value.Value.Default;
 import org.immutables.value.Value.Derived;
 
 import java.math.BigInteger;
+import java.time.Instant;
 import java.util.Base64;
 import java.util.Objects;
 
 /**
- * An extension of {@link InterledgerPreparePacket} that can be used as an IL-DCP request over
- * Interledger.
+ * An extension of {@link InterledgerPreparePacket} that can be used as an IL-DCP request over Interledger.
  */
 public interface IldcpRequestPacket extends InterledgerPreparePacket {
 
+  InterledgerAddress PEER_DOT_CONFIG = InterledgerAddress.of(InterledgerAddressPrefix.PEER.with("config").getValue());
+
   InterledgerCondition EXECUTION_CONDITION = InterledgerCondition.of(
```

Comment:
So basically `new byte[32]`

Reply:
Yep, though I _suppose_ what is currently there is [a bit more explicit](https://github.com/interledger/rfcs/blob/master/0031-dynamic-configuration-protocol/0031-dynamic-configuration-protocol.md#packet) to show spec-conformance, and the hashing is only done once in the JVM so not hurting performance.

Reply:
Ah, ok, might be nice to have a named constant then. I personally can't read base64 :)
I see elsewhere comments about what it is:
```
  /**
   * The execution_condition of an ILP packet for IL-DCP is always	   * The execution_condition of an ILP packet for IL-DCP is always
   * <tt>Zmh6rfhivXdsj8GLjp+OIAiXFIVu4jOzkCpZHQ1fKSU=</tt> in Base64 format.	   * <tt>Zmh6rfhivXdsj8GLjp+OIAiXFIVu4jOzkCpZHQ1fKSU=</tt> in Base64 format, which is the SHA-256 hash of a 32-byte
   * array with all 0 values.
   */	   */
```

Of course not really related to this pull request. Just commenting as I was scanning